src/main/scala/uk/gov/nationalarchives/notifications/decoders/SNSNotifyDecoder.scala

```diff
@@ -1,6 +1,6 @@
 package uk.gov.nationalarchives.notifications.decoders
 
-object GovUkNotifyKeyRotationDecoder {
+object SNSNotifyDecoder {
```

Comment:
Would you be able to rename this? The other classes don't have SNS in their names and Notify is from GovUK notify. Maybe something more generic like `ParameterStoreExpiryEvent` or something like that. What do you think?

Reply:
Yeah, that makes sense. I renamed it as you suggested.